### PR TITLE
grc: Added copyright holder to Options block

### PR DESCRIPTION
**Pull request by haakov**
As suggested in https://github.com/gnuradio/gnuradio/issues/1704.

### CODE REVIEW

File: grc/core/generator/Generator.py [hunk 1]
```diff
@@ -246,6 +247,7 @@ def uses_var_id():
             'msgs': msgs,
             'generate_options': self._generate_options,
             'callbacks': callbacks,
+            'version': gr.version
```
Comment (skoslowski) [inline grc/core/generator/Generator.py]: GRC has its own version field. Check-out grc/core/Config.py
This is, of course, the same as gr.version, but it serves a the single source within grc code.
Please use that instead

File: grc/core/generator/flow_graph.tmpl [hunk 2]
```diff
@@ -15,22 +15,31 @@
...
 #end if
+#if $flow_graph.get_option('copyright')
+# Copyright: $flow_graph.get_option('copyright')
+#end if
 #if $flow_graph.get_option('description')
 # Description: $flow_graph.get_option('description')
+$SINGLE_HASH
 #end if
 # Generated: $time.ctime()
+# Generated using GRC version $version()
```
Comment (skoslowski) [inline grc/core/generator/flow_graph.tmpl]: to match style with the other fields, I'd prefer 'GNU Radio version: ...' instead.

**Comment by haakov** [comment]
Sure! Fixed :+1:

**Comment by dkozel** [comment]
@marcusmueller This PR is ready to be merged with your approval.

**Comment by marcusmueller** [comment]
Thanks, everyone!

**Pull request merged**